.github/workflows/test.yml

```diff
@@ -1,15 +1,7 @@
 name: Test Python Script
 
 on:
-  push:
-    branches:
-      - rapid-inflows
-  pull_request:
-    branches:
-      - rapid-inflows  
-  workflow_dispatch:
-    branches:
-      - rapid-inflows  
+  [push, pull_request, workflow_dispatch]
```

Comment:
i would be ok with on pull request or workflow_dispatch. Do you think we need to do it on every push?

Reply:
I don't think it is neccesary. It's nice since it automatically tests every push, but workflow_dispatch could make it obsolete since we can run the test whenever we want. If you want me to remove it I can